### PR TITLE
replaced dbg with log

### PR DESCRIPTION
**Pull request by ineiti**

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -120,7 +120,7 @@ func verifyFile(c *cli.Context) error {
 	if len(c.Args().First()) == 0 {
 		printErrAndExit("Please give the 'msgFile'", 1)
 	}
-	dbg.SetDebugVisible(c.GlobalInt("debug"))
+	log.SetDebugVisible(c.GlobalInt("debug"))
```
Comment (liamsi) [inline client.go:123]: It's kind of confusing to have `log` and `debug` and `SetDebug`. What is `log`: a debugging utility package? Or a logging framework?

Comment (ineiti) [inline client.go:123]: Please think about this issues before you ask me to merge `ui` and `dbg` in one package called `log`. Propose a better name, please.

Comment (liamsi) [inline client.go:123]: This doesn't really have to do with `ui` and  `dbg`; it's about logging and debug output. Also I was always in favour of not creating our own logging framework in the first place.

**Comment by liamsi** [comment]
I think we really messed up two things here: What started as a debugging helper/util library is now a (almost) full fledged logging framework. And it's not clear to see the separation/reasoning between the debugging util library and the logging framework. As on can see from https://github.com/dedis/cosi/pull/38/files#r68057098 and 
https://github.com/dedis/cothority/pull/441#discussion_r67843552 and from these 2 issues https://github.com/dedis/cothority/issues/444 https://github.com/dedis/cothority/issues/191

**Comment by ineiti** [comment]
Please propose solutions.

**Comment by nikkolasg** [comment]
The issue is open in https://github.com/dedis/cothority/issues/444 let's have the discussion there gentlemen :)

**Comment by liamsi** [comment]
> Please propose solutions.

Ideally, I would use go's standard logging framework (or a third party lib) for logging (which has options to write to different files per module/package, provides log rotation and a lot of other features people would expect from a server application) 

Regarding debug output: It would be ideal, if the debug output would only appear if the the everything was compiled with a certain build flag. Referencing https://github.com/dedis/cothority/issues/444 and let the dicussion continue there.

**Pull request merged**